README.md

```diff
@@ -6,7 +6,7 @@ This project contains Kubernetes manifests for self-deployed OTLP ingest on Kube
 
 Before we begin, set required environment variables:
 ```console
-export GCLOUD_PROJECT=<your project id>
+export PROJECT_ID=<your project id>
 export PROJECT_NUMBER=<your project number>
```

Comment:
Would it make sense to help the user get this such as by initializing it from the value of the project ID? (e.g. via `gcloud projects describe ${PROJECT_ID} --format=value(projectNumber)`)?